toolkit/tools/grapher/grapher.go

```diff
@@ -66,8 +82,29 @@ func main() {
 		logger.Log.Panic(err)
 	}
 
+	var cloner *rpmrepocloner.RpmRepoCloner = nil
+	/*
+		disableUpstreamRepos is set to false because we want to download packages from upstream
+	*/
+	if *resolveCyclesFromUpstream {
+		cloner, err = rpmrepocloner.ConstructCloner(*outDir, *tmpDir, *workerTar, *existingRpmsDir, *existingToolchainRpmDir, *tlsClientCert, *tlsClientKey, *repoFiles)
+		if err != nil {
+			logger.Log.Panic(err)
+		}
+		/*setting repoFlags in cloner usePreviewRepo, disableUpstreamRepos=false*/
```

Comment:
Nit-pick: I think you already made sure we don't disable upstream repos inside `pkggen.mk` and we no longer even touch the upstream repos' flag here, so we can probably clean-up the comments.
```suggestion
	if *resolveCyclesFromUpstream {
		cloner, err = rpmrepocloner.ConstructCloner(*outDir, *tmpDir, *workerTar, *existingRpmsDir, *existingToolchainRpmDir, *tlsClientCert, *tlsClientKey, *repoFiles)
		if err != nil {
			logger.Log.Panic(err)
		}
```

And if the comments are still needed, please use `//` to keep the conventions from the other files.